packages/extra/monitoring/values.yaml

```diff
@@ -17,6 +17,14 @@ metricsStorages:
   storage: 10Gi
   storageClassName: ""
 
+## @param metricsStorages [array] Configuration of logs storage instances
+##
+logsStorages:
+- name: generic
+  retentionPeriod: "1m"
+  storage: 10Gi
+  storageClassName: replicated
```

Comment:
By default it uses `replicated` storageClass, due to fact vlog can't run as cluster for now